test/dyncfg/mzcompose.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import json
+import os
+from pathlib import Path
+from textwrap import dedent
+
+from materialize.mzcompose.composition import Composition, Service
+from materialize.mzcompose.services.materialized import Materialized
+from materialize.mzcompose.services.testdrive import Testdrive
+
+SERVICES = [
+    Testdrive(),
+    Materialized(),
+]
+
+
+def workflow_default(c: Composition) -> None:
+    # Create config file in MZ_ROOT directory
+    mz_root = Path(os.environ.get("MZ_ROOT", Path(__file__).parent.parent.parent))
+    config_file = mz_root / "dyncfg-test-config.json"
```

Comment:
This also doesn't match the path in the either the CRD docs or the materialized.py mzcompose service file.

Reply:
Yeah, this doesn't have to match the CRD, arguably it's a better test if it doesn't. We pass this config file path through to the Materialize mzcompose service and it sets the flag (via env vars) directly. As long as the file is accessible in the container and the path is correct everything should be good.